shared/deployment/secrets/_default.md

```diff
@@ -0,0 +1,110 @@
+### Build Time only Secret File
```

Comment:
This currently only works for `balena push` on the remote builders. LocalMode push and `deploy` don't work.